src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  transition: transform 0.3s linear;
+
+  &:hover {
+    transform: scale(1.2);
+    transition: 0.3s;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: $main-color;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__rewiews {
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-label {
+    font-size: 12px;
+    line-height: 18px;
+    color: #616070;
+  }
+
+  &__price-amount {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__button {
+    box-sizing: border-box;
+    display: inline-block;
+    background: $active-color;
+    color: #fff;
+    border-radius: 5px;
+    font-weight: 700;
+    font-size: 14px;
+    height: 40px;
+    text-align: center;
+    line-height: 16px;
+    text-transform: uppercase;
+    text-decoration: none;
+    border: 1px solid $active-color;
+    padding: 12px 70px;
+    transition: 0.3s;
+
+    &:hover {
+      background-color: #fff;
+      color: $active-color;
+      cursor: pointer;
+      transition: 0.3s;
```

Comment:
redundant

Reply:
sorry, i didn't get correctly. If button background-color was #00acdc,  but on hover it has to be change on  #fff. How correctly to write this?
I fixed for `background-color: rgba($color: $active-color, $alpha: 0);`. Is it ok?